doc/news/_preparation_next_release.md

```diff
@@ -183,6 +183,14 @@ you up to date with the multi-language support provided by Elektra.
 - Improved documentation for module keytest in Elektra Core. _(@lawli3t)_
 - Improved documentation for module keyset in Elektra Core. _(@lawli3t)_
 - Fixed example in the "mount-configuration-files" tutorial [#3722](https://github.com/ElektraInitiative/libelektra/issues/3722). _(Philipp Oppel)_
+- Reworked [METADATA.ini](/doc/METADATA.ini) _(Markus Raab)_
+- Write notes that `\\` are due to shell recorder, and are not to be copied _(Markus Raab)_
+- Add link to [Go](https://github.com/ElektraInitiative/go-elektra) bindings _(Markus Raab)_
+- Fix order of tutorials _(Markus Raab)_
+- Added API-Reviews for multiple functions in the public API _(Stefan Hanreich)_
+- Minor rewording in [java-kdb.md](/doc/tutorials/java-kdb.md) _(@aaronabebe)_
+- Added interactive and non-interactive versions of a new tutorial to writing a specification _(@aaronabebe)_
+- Add a short Visual Studio 2019 tutorial _(Dominic Jäger)_
```

Comment:
Seems like some unrelated lines appeared.
```suggestion
- Added interactive and non-interactive versions of a new tutorial to writing a specification _(@aaronabebe)_
```

Reply:
Thanks for pointing this out! It also looks like doc/news/_preparation_next_release.md now has a merge conflict, so a rebase would be useful anyway.

@aaronabebe you might also want to read doc/GIT.md "Working with forks" and after pulling you do `git rebase master` on your branch.

Reply:
Seems like something went wrong with the rebase, now there are even more changes done in other PRs. It is probably best if you cherry-pick your commits in a new branch.

Reply:
@markus2330 yeah i don't know what happened. so how would i go about doing that? because i was lazy and just pushed to the `master` of my fork. 

So revert my fork to `upstream/master` and then reapply the commits in a new branch of my fork and redo from that? 
Or should I just open a new PR from a new branch? 
What would be best here? 

Sorry for the inconvinience.

Reply:
Just in case, make a backup of your whole repository, then:

1. Write down the list of commits that are relevant for this PR.
2. Hard reset your master to libelektra master (probably origin/master)
3. git cherry-pick <the list you have from 1.>
4. push --force